server/serverMethods.js

```diff
@@ -61,7 +61,27 @@ function getUserCountry(req) {
 
 const securityHeaders = helmet({
   referrerPolicy: { policy: 'no-referrer-when-downgrade' },
-  contentSecurityPolicy: false,
+  contentSecurityPolicy: {
+    useDefaults: true,
+    directives: {
+      defaultSrc: ["'self'", "'unsafe-inline'"],
```

Comment:
I would advise not to use naked asterisks (*)  or "unsafe-inline" where possible. Those are some of the line items that we addressed on ID-Server security review https://jira.vonage.com/browse/VIDECO-9976

Reply:
I see. I have removed "'unsafe-inline'" from the `defaultSrc` directive. But I found that I had to leave "'unsafe-inline'" in place for the `scriptSrc` (which is normal, I think) and `styleSrc` directives. However [VIDECO-9976](https://jira.vonage.com/browse/VIDECO-9976) asks us not to set it for `styleSrc`. But I will leave it in, because without it, OpenTok.js videos are badly misaligned, and you get an error:

```Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' cdnjs.cloudflare.com assets.tokbox.com static.opentok.com https://unpkg.com/@vonage/video-client@2/dist/js/opentok.js https://unpkg.com/@vonage/client-sdk-video@2/dist/js/opentok.js". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution.
```

This is because OpenTok.js loads an inline style (from the WebPack's style-loader  module):

```
var update = require("!../../node_modules/style-loader/lib/addStyles.js")(content, options);
```

Reply:
Got it. My understanding is that it's not a best practice to have 'unsafe-inline' in the script-src. hmmmmm...a common workaround would be to move the javascript into its own file-but these files aren't static.

Reply:
There's no JavaScript file that we can move here (in this project). The script that loads the styles inline is from WebPack's style-loader, used in OpenTok.js. Not in the source code for this project.